microsetta_interface/templates/sample_results.jinja2

```diff
@@ -0,0 +1,104 @@
+{% extends "sitebase.jinja2" %}
+{% set page_title = "Sample Results" %}
+{% set show_breadcrumbs = True %}
+{% set show_logout = True %}
+
+{% block head %}
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.21/css/jquery.dataTables.css" />
+  <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.21/js/jquery.dataTables.js"></script>
+
+  <!-- vega header(s) -->
+  <script src="https://cdn.jsdelivr.net/npm/vega@5"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vega-lite@4"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vega-embed@6"></script>
+
+  <link rel="stylesheet" type="text/css" href="/static/css/4_column_flex.css" />
+  <script>
+    let taxonomy = "{{taxonomy}}";
+    let metric = "{{alpha_metric}}";
+    let sampleId = "{{sample.sample_id}}";
+    var opt = {"renderer": "canvas", "actions": true};  /* Options for the Vega embedding */
+
+    function createDataTable() {
+        return function() {
+            $('#taxonomyTable').DataTable({
+                ajax: {
+                    url: "{{public_endpoint}}/taxonomy/present/single/" + taxonomy + "/" + sampleId
```

Comment:
Barcodes in Microsetta do not contain the Qiita study prefix. I'm not seeing anywhere that the `10317.` is being prepended?

Reply:
Yeah... Also we did not pass the correct variable to `sampleId`. Rought draft 😅

Reply:
Updated in https://github.com/biocore/microsetta-interface/pull/11/commits/fd7d375a6ac31e7cf08bace8f770b65c53089b37